hadoop-hdds/container-service/pom.xml

```diff
@@ -177,6 +168,11 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <groupId>org.apache.ratis</groupId>
       <artifactId>ratis-proto</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>org.apache.ratis</groupId>
+      <artifactId>ratis-server</artifactId>
```

Comment:
For future patches: I think we can remove empty lines in `dependencies` section.  Especially where the items before/after the empty line are from the same group.

Reply:
Thanks for the catch. I missed this while reviewing the output.
> Especially where the items before/after the empty line are from the same group.

We can always remove the empty lines to make it consistent.